internal/host/host.go

```diff
@@ -51,6 +49,16 @@ var manualRebootStages = [...]models.HostStage{
 	models.HostStageDone,
 }
 
+var InstallationProgressTimeout = map[models.HostStage]time.Duration{
+	models.HostStageStartingInstallation: 10 * time.Minute,
+	models.HostStageWritingImageToDisk:   20 * time.Minute,
+	models.HostStageRebooting:            30 * time.Minute,
+	models.HostStageConfiguring:          30 * time.Minute,
+	models.HostStageWaitingForIgnition:   60 * time.Minute,
+	models.HostStageInstalling:           20 * time.Minute,
+	"DEFAULT":                            60 * time.Minute,
```

Comment:
Probably should be a const.

Reply:
Actually i'm not sure that we should have a default. we probably want to fail if we have something unexpected.